docs/rfcs/2024-12-06-chat-tags.md

```diff
@@ -0,0 +1,25 @@
+# Chat tags, emoji entry
+
+## Problem
+
+Organize chats.
+
+Stop users from using chat profiles to organize chats.
+
+More fun as we solve groups problem.
+
+## Solution
+
+This is a simple "quality of life" improvement that's long due, as the number of users and conversations grows, that would probably take just a week to ship.
+
+Proposed UX:
+- tags are per profile.
```

Comment:
To add a new tag do I need to tag someone first? Given the example of "Family" tag I most likely have a few family members (20, 30) would I need to tag them 1 by 1? It would be better UX if we could create a new tag by click some + button and in that case be able to select a range of contacts to add to that tag

Reply:
let's not complicate. There is an argument for multi-select in the list, but seems a separate feature